### PR TITLE
Fix compilation

### LINKED ISSUE #6: Compilation issues
There are some syntax issues making typings invalid

### PR DESCRIPTION
**Pull request by mnaoumov**
Fixes #6

### CODE REVIEW

**Comment by mnaoumov** [comment]
@Fevol don't see your comments either

**Review by Fevol — COMMENTED** [review]
Questions about potentially unused import and missing typeof

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Fevol/obsidian-typings/issues"
+  },
+  "homepage": "https://github.com/Fevol/obsidian-typings#readme",
+  "dependencies": {
+    "obsidian": "^1.4.11"
+  },
+  "devDependencies": {
+    "@ophidian/electron-types": "^24.3.1"
```
[the remark this answers is not in the document]
Comment (mnaoumov) [reply package.json:26]: Yes, it is used for https://github.com/Fevol/obsidian-typings/pull/7/files#diff-af0bf9c1bf3f88c2d78b2384c61f96bbed0d849ceaa9a0acdfb71b933badcf4dR26

File: obsidian-ex.d.ts [hunk 2]
```diff
@@ -2559,7 +2532,7 @@ declare module 'obsidian' {
 		/**
```
[the remark this answers is not in the document]
Comment (mnaoumov) [reply obsidian-ex.d.ts:2532]: No, with `typeof` it won't work. The difference is caused by different `import` formats for `IpcRender` and `fs`

**Pull request merged**